css/style.css

```diff
@@ -0,0 +1,64 @@
+
+
+:root {
+    --blue: rgb(92, 92, 207);
+    --background: rgb(214, 211, 232);
```

Comment:
For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡